starknet_py/constants.py

```diff
@@ -39,14 +39,6 @@
 
 ROOT_PATH = Path(__file__).parent
 
-# Ledger constants
-STARKNET_CLA = 0x5A
-EIP_2645_PURPOSE = 0x80000A55
-EIP_2645_PATH_LENGTH = 6
-PUBLIC_KEY_RESPONSE_LENGTH = 65
-SIGNATURE_RESPONSE_LENGTH = 65
-VERSION_RESPONSE_LENGTH = 3
-
```

Comment:
note: These variables aren't used across multiple files so let's keep them in the dedicated one.